src/components/variable.jsx

```diff
@@ -48,51 +48,127 @@ const Variable = React.createClass({
+      ETERNITY: <FormattedMessage id='forEternity'/>
     }
+    const valueTypeMessage = {
+      Int: <FormattedMessage id='anInteger'/>,
+      Float: <FormattedMessage id='aFloat'/>,
+      Date: <FormattedMessage id='aDate'/>,
+    }
+    function formatDefaultValue(variable){
+    if (variable.valueType == "Date"){
+      return <FormattedDate value={variable.defaultValue} year='numeric' month='2-digit' day='2-digit'/>
+    }else if(variable.valueType == "Float"){
+      if (String(variable.defaultValue).indexOf('.') != -1){
+        return String("variable.defaultValue").indexOf('.')
+      }else{
+        return String(variable.defaultValue) + '.0';
+      }
+    }else{
+      return String(variable.defaultValue)
+    }
+  }
     return (
       <div>
-        <dl>
-          <dt>Cette variable est définie pour</dt>
-          <dd>{entityMessage[variable.entity]}.</dd>
-          <dt>Elle se calcule à l'échelle</dt>
-          <dd>{definitionPeriodMessage[variable.definitionPeriod]}.</dd>
-          <dt>Elle est de type</dt>
-          <dd>{variable.valueType}.</dd>
-          <dt>Sa valeur par défault est</dt>
-          <dd>{String(variable.defaultValue)}.</dd>
-          { variable.references && <dt>Références&nbsp;:</dt> }
-          { variable.references && (
-            <dd><ul>
+        <p>
+            <FormattedMessage id='entityParagraph'
+              values={{
+                variableLink: 
+                  <a href="href='https://doc.openfisca.fr/variables.html'">
+                    <FormattedMessage id='variableText'/>
+                  </a>,
+                entityLink:
+                  <a href="https://doc.openfisca.fr/periodsinstants.html">
+                    <FormattedMessage id='entityText'/>
+                  </a>
+                }}/>
+            <span className="variableMetadataHighlight">
+              {entityMessage[variable.entity]}
+            </span>
+          .
+        </p>
+        <p>
+          <FormattedMessage id='definitionPeriodParagraph' 
+            values={{definitionPeriodLink: 
+              <a href="https://doc.openfisca.fr/periodsinstants.html">
+              <FormattedMessage id='definitionPeriodText'/></a>}}
+            />
+          <span className="variableMetadataHighlight">
+            {definitionPeriodMessage[variable.definitionPeriod]}
+          </span>
+          .
+        </p>
+        <p>
+          <FormattedMessage id='valueTypeParagraph'/> 
+          <span className="variableMetadataHighlight">
+            {variable.valueType in valueTypeMessage ? (
+              <span> {valueTypeMessage[variable.valueType]}.</span>
+            ) : (
+              <span> {variable.valueType}.</span>
+            )}
+          </span>
+        </p>
+        <p>
+          <FormattedMessage id='defaultValueParagraph'
+            values={{defaultValueLink:
+              <a href="https://doc.openfisca.fr/coding-the-legislation/20_input_variables.html">
```

Comment:
Yes, but for a variable such as `rsa`, it's not the case: if I provide no value, the default value won't be used, OpenFisca will run a computation to calculate the `rsa` !